packet/templates/packet.html

```diff
@@ -1,32 +1,30 @@
 {% extends "extend/base.html" %}
 
-{% set packet_end = freshman.current_packet().end.strftime('%m/%d/%Y') %}
+{% set packet_end = packet.end.strftime('%m/%d/%Y') %}
```

Comment:
If possible, complicated logic should be handled in the controller, not the template.

Reply:
yeah, that's my preference too but do to the pending UI rebuild I tried to stay away from putting too much work into the templates in this PR.